.coderabbit.yaml

```
@@ -0,0 +1,4 @@
+reviews:
+  path_filters:
+    ignored:
+      - "**/vendor/**"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)